TA_Scheduler_App/courseFeatures.py

```diff
@@ -50,30 +61,33 @@ def delete_section(sectionKey:int):
     #         return False
 
     @staticmethod
-    def edit_course(courseKey:int, newCourseName:str=""):
+    def edit_course(courseKey:int, newCourseName:str="",newSemester:str = ""):
         try:
             course = Course.objects.get(pk=courseKey)
         except Course.DoesNotExist:
             return False
-
-
-        if newCourseName is not '' and newCourseName is not None:
-            course.courseName = newCourseName
+        if newCourseName and newCourseName.strip():
+            course.courseName = newCourseName.strip()
+            course.save()
+        if newSemester and newSemester.strip():
+            course.semester = newSemester.strip()
             course.save()
 
         return course
 
     @staticmethod
-    def edit_section(sectionKey:int, newSectionTime:str=None):
+    def edit_section(sectionKey:int, newSectionCode: str,newInstructor: str):
```

Comment:
```suggestion
    def edit_section(sectionKey: int, newSectionCode: str = "", newInstructor: str = ""):
```
for consistency with `edit_course` **_unless_** we want to allow for either `sectionCode` or `instructor` to be blank for sections.